### PR TITLE
fix(TDOPS-3511/locales): add component type argument to the codeshift

### PR DESCRIPTION
**Pull request by ybaskaran**
**What is the problem this PR is trying to solve?**
Currently there is no option to run the codemod on specific i18n expression (t, trans)

**What is the chosen solution to this problem?**
add component type argument to the codeshift

**Please check if the PR fulfills these requirements**

- [x] The PR have used `yarn changeset` to a request a release from the CI if wanted.
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: tools/scripts-locales/codeshift/locales.js [hunk 1]
```diff
@@ -128,9 +128,15 @@ export default function transformer(fileInfo, api, options) {
 	try {
 		const en = getLocales(options.ref);
 		const j = api.jscodeshift;
-		const root = j(fileInfo.source);
-		let result = searchAndUpdateI18nValues(j, root, en);
-		if (result) {
+		options.comp = options.comp === 'all' ? ['i18n', 'trans'] : options.comp;
```
Comment (jmfrancois) [inline tools/scripts-locales/codeshift/locales.js:131]: it looks like options.comp is undefined by default.
Could you please make it 'all' by default

File: tools/scripts-locales/README.md [hunk 2]
```diff
@@ -4,11 +4,13 @@ This package expose a command line useful to manage locales.
 
 Commands:
 
-- `npx talend-locales update-code --ref ./node_modules/@talend/locales-tui-components/locales/en ./packages/components/src/` will update default message inside source by using english translations (ref).
+- `npx talend-locales update-code --ref ./node_modules/@talend/locales-tui-components/locales/en --comp=i18n ./packages/components/src/` will update default message inside source by using english translations (ref).
 
 ```
 args[0] => reference -> locales_en.json or the folder with multiple locale json (for ex: settings.json, app.json etc.,)
-args[1] => target -> folder with application source (ex: ./packages)
+args[1] => i18n component to find -> can be any of ['i18n', 'trans', all]
```
Comment (jmfrancois) [inline tools/scripts-locales/README.md:11]: document the default is 'all'
```suggestion
args[1] => i18n component to find -> can be any of ['i18n', 'trans', 'all'] 
```

**Review by jmfrancois — CHANGES_REQUESTED** [review]

File: tools/scripts-locales/codeshift/locales.js (hunk 1 above)
Comment (ybaskaran) [reply tools/scripts-locales/codeshift/locales.js]: fixed

File: tools/scripts-locales/README.md (hunk 2 above)
Comment (ybaskaran) [reply tools/scripts-locales/README.md]: fixed

**Review by jmfrancois — APPROVED** [review]

**Pull request merged**